test/distributed/test_dist_link_neighbor_sampler.py

```diff
@@ -445,21 +445,11 @@ def test_dist_link_neighbor_sampler(disjoint):
         s.bind(('', 0))
         port = s.getsockname()[1]
 
-    world_size = 2
-    w0 = mp_context.Process(
-        target=dist_link_neighbor_sampler,
-        args=(world_size, 0, port, disjoint),
-    )
-
-    w1 = mp_context.Process(
-        target=dist_link_neighbor_sampler,
-        args=(world_size, 1, port, disjoint),
-    )
-
-    w0.start()
-    w1.start()
-    w0.join()
-    w1.join()
+    procs = (mp_context.Process(target=dist_link_neighbor_sampler,
+                                args=(0, port, disjoint)),
```

Comment:
missing world_size=2 at the beginning of the args? Same for other cases.

Reply:
world size is calculated in `dist_util.py:59` from number of elements of procs parameter:
`world_size = len(procs)`

Reply:
So we can remove it from the function arguments?

Reply:
`world_size` is used i.e. in the above code in `dist_link_neighbor_sampler` function, so it's still needed. However calculation of it later in the assert_run_proc makes it easiy to pass any number of processes without the need for remembering to pass it as the first parameter for the original target function